maui-toc.html

```diff
@@ -1287,6 +1291,7 @@
             <li><a href="/maui/TimePicker/time-interval">Time Interval</a></li>
             <li><a href="/maui/TimePicker/customizations">Customizations</a></li>
             <li><a href="/maui/TimePicker/events">Events</a></li>
+			<li><a href="/maui/TimePicker/commands">Commands</a></li>  
```

Comment:
As per discussion move commands in events topic

Reply:
Commands topic moved under Events.